css-three-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+* {
+  box-sizing: border-box;
+  font-family: Lato, sans-serif;
+}
+
+.container {
+  max-width: 1000px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.column-one-third {
+  width: calc(100% / 3);
+  position: relative;
+}
+
+body {
+  background-color: #3696da;
+  margin: 40px;
+  padding: 20px;
+}
+
+h1,
+h3 {
+  text-align: center;
+  font-weight: lighter;
+  color: white;
+}
+
+.icon {
+  z-index: 1;
+  height: 55px;
+  width: 55px;
+  position: absolute;
+  top: 28px;
+  left: 130px;
+}
```

Comment:
You can avoid using `position: absolute` here by making the `img` a child of the `div` you made into the circle.